java/core/src/test/bond/bonded.bond

```diff
@@ -2,6 +2,10 @@ import "common.bond"
 
 namespace org.bondlib.test
 
+struct BondedBase {
```

Comment:
Consider "HasBondedField" and "bondedField" instead. The base/derived part isn't relevant here, but "BondedBase" makes me think that this is a test of inheritance.